app/javascript/stylesheets/_email.scss

```diff
@@ -18,10 +18,32 @@ $accent-color: map-get($colors, "pink"); // [1]
     max-width: $main-stack-width;
 
     table {
-      // [5]
-      gap: 0;
-      tr:not(:last-child) {
-        margin-bottom: spacer(4);
+      &,
```

Comment:
Added back the minimum styles to make emails work - they're fixed width so responsive styling isn't needed. As I mentioned in your PR, I think we can do this for now and look into options for email styling separately